javascript/347-Top-K-Frquent-Elements.js

```diff
@@ -6,7 +6,7 @@
 var topKFrequent = function(nums, k) {
     let map = new Map();
     let res = [];
-    let bucket = new Array(nums.length+1);
+    let bucket = new Array(nums.length+1).fill(null).map(() => []); // to create unique arrays
```

Comment:
Could be `new Array(nums.length+1).fill().map(() => []);` or `Array.from({length: nums.length+1}, () => [])`